lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -71,6 +72,34 @@ def self.create_release(repository, version, release_notes, assets, prerelease)
           github_client().upload_asset(release[:url], file_path, content_type: 'application/octet-stream')
         end
       end
+
+      # Downloads a file from the given GitHub release
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] relase The title of the GitHub release
+      # @param [String] file_path The path, inside the project folder, of the file to download
+      # @param [Striog] download_folder The folder which the file should be downloaded into
+      # @return [String] The path of the downloaded file, or nil if something went wrong
+      #
+      def self.download_file_from_release(repository:, release:, file_path:, download_folder:)
+        repository = repository.delete_prefix('/').chomp('/').concat('/')
+        file_path = file_path.delete_prefix('/').chomp('/').concat('/')
+        release = release.concat('/')
+        file_name = File.basename(file_path)
+        download_path = File.join(download_folder, file_name)
+
+        begin
+          open(URI.join('https://raw.githubusercontent.com/', repository, release, file_path).to_s.chomp('/')) do |remote_file|
+            open(download_path, 'wb') do |file|
+              file.write(remote_file.read)
+            end
```

Comment:
I _think_ you can simplify that ysing `File.write(dest, content)` ie:

```suggestion
          File.write(download_path, remote_file.read)
```